pixeltable/exprs/arithmetic_expr.py

```diff
@@ -59,7 +76,12 @@ def _id_attrs(self) -> list[tuple[str, Any]]:
         return super()._id_attrs() + [('operator', self.operator.value)]
 
     def sql_expr(self, sql_elements: SqlElementCache) -> Optional[sql.ColumnElement]:
-        assert self.col_type.is_int_type() or self.col_type.is_float_type() or self.col_type.is_json_type()
+        assert (
```

Comment:
Is something like 'string_col * 5' even supported by Postgres?

Reply:
https://www.w3resource.com/PostgreSQL/repeat-function.php